### ISSUE TITLE
Null ref: T2.Projectile.MakeBoomerangAboutFaceOrKeepGoingForMultishot

### ISSUE THREAD

**Issue by ChrisNZL** [ChrisNZL/Tallowmere2#496]
Auto bug report from 0.1.3.
Feedback IDs:
- 20200423-XFMTY
- 20200423-QGT53

Method is failing to get a Unity Transform due to polling within FixedUpdate.

```
3:23:12, Frame 57131, EXCEPTION »  NullReferenceException
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

UnityEngine.Transform.get_position ()
T2.Projectile.MakeBoomerangAboutFaceOrKeepGoingForMultishot ()
T2.Projectile.FixedUpdate ()

GameStates: RompingThroughDungeon
GameSetupMode: Singleplayer
Players: 1
Room: 47 / 47
RoomModifiers: Spiky (Spiky), NoHealing (No Healing), MoreGoldOnDeath (Wealth), Bouncy (Bouncy), BloatsOnly (Bloat Party)
```

**Comment by ChrisNZL** [comment]
Fixed in 0.1.3c.

**Comment by ChrisNZL** [comment]
Further auto bug report from 0.1.4b.

Need an additional try/catch – attempting to get a `Transform` within FixedUpdate.

Feedback ID: 20200630-QB6FV

```
7:40:43, Frame 43359, EXCEPTION »  NullReferenceException
>>>>> CRITICAL ERROR >>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>>

UnityEngine.Transform.get_position ()
T2.Projectile.MakeBoomerangAboutFaceOrKeepGoingForMultishot ()
T2.Projectile.FixedUpdate ()

GameStates: RompingThroughDungeon
GameSetupMode: Singleplayer
Players: 1
DungeonRoomTransitionerState: Null
Room: 7 / 7
RoomModifiers: ElementalBallsOnly (仅限喷吐攻击)
YouHaveDiedState: Null
```

**Comment by ChrisNZL** [comment]
Should be fixed in 0.1.4d.
